core/src/main/java/org/kiwix/kiwixmobile/core/main/WebViewCallback.java

```diff
@@ -35,4 +36,6 @@
   void webViewPageChanged(int page, int maxPages);
 
   void webViewLongClick(String url);
+
+  void onFullscreenVideoToggled(Boolean isFullScreen);
```

Comment:
This is still the boxed type `Boolean`, is there some issue making it the primitive `boolean`

Reply:
Sorry, @macgills .I didn't know that this exists in java also I didn't read the review so carefully.

Reply:
no problemo